exp-player/addon/styles/components/exp-overview.scss

```diff
@@ -0,0 +1,5 @@
+.format-average-label {
```

Comment:
I continue to be worried about all the special-case styles for each individual element- especially since these are pixel-specific placements.

What happens when you change the default font size, zoom the page, resize the browser, or translate this string to a longer/shorter word? Does the label still appear centered in the way you expect?